### ISSUE TITLE
LLM(Large Language Model)の学習まとめ

### ISSUE THREAD

**Issue by nariaki3551** [nariaki3551/library#64]

**Comment by nariaki3551** [comment]
### PaLM (#63)

GoogleのTPUv4で学習された540億パラメタモデル

We train all three models on exactly one epoch of the data (shuffled identically for all models) and choose the mixing proportions to avoid repeating data in any subcomponent.
> 我々は3つのモデル全てを正確に1エポックのデータで訓練し（全てのモデルで同じようにシャッフルされる）、どのサブコンポーネントでもデータの繰り返しを避けるように混合比率を選択する。

**Comment by nariaki3551** [comment]
## LaMA (#65)

学習に用いたデータセットとその割合、エポック数

<img src=https://github.com/nariaki3551/library/assets/21208556/9f4166b1-3257-4f10-b061-c468a435a339, width=400>

WikipediaとBooksは2エポック、それ以外は1エポックになるようにサンプリング割合を調整した模様

Overall, our entire training dataset contains roughly 1.4T tokens after tokenization. For most of our training data, each token is used **only once** during training, with the exception of the Wikipedia and Books domains, over which we perform approximately **two epochs**.
